package.json

```
@@ -15,7 +15,7 @@
     "lint": "yarn lint:css && yarn lint:js && yarn lint:markdown",
     "lint:css": "stylelint \"packages/*/src/**/*.js\" && stylelint \"utils/**/*.js\"",
     "lint:js": "eslint packages/*/src/ utils/ --max-warnings 0",
-    "lint:markdown": "markdownlint README.md packages/*/src/**/*.md packages/*/src/*.md packages/*/README.md",
+    "lint:markdown": "markdownlint API.md DEVELOPMENT.md README.md VERSIONING.md packages/*/src/**/*.md packages/*/src/*.md packages/*/README.md",
```

Could we just lint all root `*.md` files?
```suggestion
    "lint:markdown": "markdownlint *.md packages/*/src/**/*.md packages/*/src/*.md packages/*/README.md",
```

---

We don’t want to include CHANGELOG.md